.circleci/config.yml

```diff
@@ -240,27 +240,47 @@ jobs:
     resource_class: 2xlarge+
     steps:
       - checkout
+      - run:
+          name: Generate cache key
+          # This will refresh cache on Sundays, nightly build should refresh the cache.
+          command: echo "$(date +"%Y-%U")" > .circleci-weekly
```

Comment:
We could use this as an opportunity to verify that the download functions continue to download the same files.